Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG NODE_VERSION=10.15.2
```

Comment:
Are you using this node version in 6.7.2?

Reply:
Node.js v10.15.2 is the version specified in the `.node-version` file.